Syntaxes/RSpec.tmLanguage

```diff
@@ -92,7 +94,7 @@
 		<key>behaviour</key>
 		<dict>
 			<key>begin</key>
-			<string>^\s*([fx]?(?:describe|context|feature))\b(?=.*\bdo\s*$)</string>
+			<string>^\s*([fx]?(?:RSpec\.describe|describe|context|feature))\b(?=.*\bdo\s*$)</string>
```

Comment:
Had a second look at how this is dealing with the leading `RSpec.` and fc9d4cf should be a definite improvement. Fixes all the occurences and the symbol list.